client/web/compose/src/components/Namespaces/Reminders/List.vue

```diff
@@ -11,7 +11,7 @@
         <hr v-if="r.dismissedAt && sortedReminders[i - 1] ? !sortedReminders[i - 1].dismissedAt : false ">
 
         <div
-          class="overflow-auto border card shadow-sm my-2 p-1"
+          class="border card shadow-sm my-2 p-1"
```

Comment:
Are you sure we can just remove the overflow-auto? Pretty sure it was there for a reason